R/create_service_use_lookup.R

```diff
@@ -0,0 +1,330 @@
+#' Title Create the Service Use Cohort lookup file
+#'
+#' @param data A data frame
+#' @param year The financial year
+#' @param write_to_disk Default [TRUE]
+#'
+#' @return Either save out a copy of the lookup, or return it for checks
```

Comment:
The `@return` value is what is returned by the function. So in this case that should always be the data. Whether the data gets written to the disk or not is a 'side effect' of the function, which should be documented in the `write_to_disk` param.

Reply:
Cool, thanks for the tip